### DISCUSSION TITLE
Setting biome to specific area

### DISCUSSION THREAD

**Discussion by Davisiiiik** [BG-Software-LLC/SuperiorSkyblock2#885] [Pending Feature Requests]
In previous skyblock plugin I used (uSkyBlock) was a feature to set a biome to specific area around the player:
- Given the blocks distance (ie: 20 blocks to every side around the player)
- Given the chunk distance (ie: 3 chunks to every side around the player)
- There was still a choice to change the biome on the whole island

This is the only feature I miss here and was better in previous plugin and my players are kinda used to it. You could have multiple biomes on your island for different types of farms (ie: guardian farm).

**Comment by matobodo** [comment]
I have looked a bit into this feature. And in my opinion, it requires the following things to be done.

- separate biome changing for each world, the biome only changes for the world where the player is currently in
- separate biome GUI for each world type - add files biomes-nether.yml, biomes-the-end.yml, and biomes-custom.yml. biomes.yml will stay for the default world for backward compatibility
- add special buttons to GUI, which will change the size setting
- rework the `getBiome` and `setBiome` methods while ensuring backward compatibility
- add support to command `/island admin setbiome` - optional?
- update `IslandBiomeChangeEvent`
- update biome placeholder in `PlaceholderHook`

I have already started working on some needed util functions.
